jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TravelerMessageBuilder.java

```diff
@@ -233,4 +252,41 @@
 //        }
 //        return area;
 //    }
-//}
+
+    private long getStartTime(TravelerInputData.DataFrame dataFrame) throws ParseException {
+        Date startDate = sdf.parse(dataFrame.startTime);
+        String startOfYearTime = "01/01/" + getStartYear(dataFrame) + " 12:00 AM";
+        Date startOfYearDate = sdf.parse(startOfYearTime);
+        long minutes = ((startDate.getTime() - startOfYearDate.getTime()) / 60000);
+        return minutes;
+    }
+
+    private int getStartYear(TravelerInputData.DataFrame dataFrame) throws ParseException {
```

Comment:
Method should use DateTimeUtils and ZonedDateTime java classes for parsing ISO date-time.